medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,17 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_primary_id = fields.Many2one(
```

Comment:
ah I see. So I guess this should be removed, and then just have something like `secondary_parent_ids` for the other field?